win/sdl2.py

```diff
@@ -1,12 +1,12 @@
 from os import walk
 from .common import *
 
-__version__ = '0.4.5'
+__version__ = '0.4.6'
```

Comment:
probably should bump to `0.5.0`.

Reply:
I thought the same this morning, but from the git blame seems we did not bump the minor version in past when updating sdl2 deps.

But.. I'm super-fine with it as it absolutely makes sense.

Reply:
Did it.

Reply:
It's because sdl2 bumped their minor version numbers, which they hadn't in the past 😅